src/SprykerEco/Yves/Billpay/Plugin/ServiceProvider/TwigBillpayServiceProvider.php

```diff
@@ -0,0 +1,256 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Yves\Billpay\Plugin\ServiceProvider;
+
+use Generated\Shared\Transfer\CountryTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+use Silex\Application;
+use Silex\ServiceProviderInterface;
+use Spryker\Shared\Config\Config;
+use Spryker\Shared\Kernel\Store;
+use Spryker\Yves\Kernel\AbstractPlugin;
+use SprykerEco\Shared\Billpay\BillpaySharedConfig;
+use SprykerEco\Shared\Billpay\BillpayConstants;
+use Twig_SimpleFunction;
+
+/**
+ * @method \SprykerEco\Yves\Billpay\BillpayFactory getFactory()
+ */
+class TwigBillpayServiceProvider extends AbstractPlugin implements ServiceProviderInterface
+{
+    const MONEY_DIVIDER = 100;
+
+    /**
+     * @param \Silex\Application $app
+     *
+     * @return void
+     */
+    public function register(Application $app)
+    {
+        $app['twig'] = $app->share(
+            $app->extend('twig', function (\Twig_Environment $twig) {
+                $twig->addFunction($this->getBillpayFunction());
+                return $twig;
+            })
+        );
+    }
+
+    /**
+     * @param \Silex\Application $app
+     *
+     * @return void
+     */
+    public function boot(Application $app)
+    {
+    }
+
+    /**
+     * @return \Twig_SimpleFunction
+     */
+    public function getBillpayFunction()
+    {
+        $quoteClient = $this->getFactory()->getQuoteClient();
+
+        return new Twig_SimpleFunction('billpay', function ($identifier) use ($quoteClient) {
+
+            if ($identifier === 'salutation') {
```

Comment:
Instead of magic string you could reuse AbstractManagerInterface constant, right?